packages/core-utils/src/map.ts

```diff
@@ -133,7 +133,7 @@ export function itineraryToTransitive(
           // OTP2 Scooter case
           leg.mode === "SCOOTER"
         ) {
-          fromPlaceId = `escooter_rent_station_${leg.from.bikeShareId}`;
+          fromPlaceId = `escooter_rent_station_${leg.from.name}`;
```

Comment:
This change contradicts https://github.com/opentripplanner/otp-ui/pull/400/files#r882102376.

Reply:
I forgot to add a comment here, which I will do. Essentially the reason that I moved back to `name` is that the fromPlace is used to generate the title of the e-scooter on the map. While `bikeShareId` is always a long yucky uuid, `name` has the potential to be a nicer title. 

The e-scooter naming is a little bugged right now, so this will likely be changed again in the future.

Reply:
Ok, but there was another instance around line 158 so both need to be consistent.

Reply:
I will undo the change for now and make a separate full e-scooter name fix PR

Reply:
Undone in 2f4508cf91057c01c2bab6f0ec75666b49d60ecd